.travis.yml

```diff
@@ -1,14 +1,13 @@
 language: java
-sudo: false
+sudo: required
```

Comment:
hmm.. this change means we won't build using container based infra .. and thus slower builds :( ..   https://docs.travis-ci.com/user/migrating-from-legacy/